### PR TITLE
Add msxml6

### PR DESCRIPTION
**Pull request by Zemogiter**
Tested on `The Sims` wineprefix and it works.

### CODE REVIEW

File: Engines/Wine/Verbs/msxml6/script.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            .checksum("5125220e985b33c946bbf9f60e2b222c7570bfa2")
+            .name("msxml6_x86.msi")
+            .get();
+    }
+    remove(this.system32directory() + "/msxml6.dll")
+    this.overrideDLL()
+        .set("native,builtin", ["msxml6"])
+        .do();
+    if (this.architecture() == "amd64") {
+    this.wizard().wait(tr("Please wait while {0} is installed...", "msxml6"));
```
Comment (ImperatorS79) [inline Engines/Wine/Verbs/msxml6/script.js]: Winetricks also removes the 64 bits dlls in that case. Also, the indentation looks wrong.

File: Engines/Wine/Verbs/msxml6/script.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    this.run(setupFile64, ["/q:a", "/c:msxml6_x64.msi /q"], null, false, true);
+    }
+    else {
+    this.wizard().wait(tr("Please wait while {0} is installed...", "msxml6"));
+    this.run(setupFile32, ["/q:a", "/c:msxml6_x86.msi /q"], null, false, true);
```
Comment (ImperatorS79) [inline Engines/Wine/Verbs/msxml6/script.js]: Same as in msxml3:
w_try_msiexec64 /i "$W_CACHE"/msxml6/msxml6_x64.msi
 w_try "$WINE" msiexec /i "$W_CACHE"/msxml6/msxml6_x86.msi $W_UNATTENDED_SLASH_Q

Comment (Zemogiter) [reply Engines/Wine/Verbs/msxml6/script.js]: I haven't tested the 64bit installation yet.

File: Engines/Wine/Verbs/msxml6/script.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    remove(this.system32directory() + "/msxml6.dll")
+    this.overrideDLL()
+        .set("native,builtin", ["msxml6"])
+        .do();
+    if (this.architecture() == "amd64") {
+        remove(this.system64directory() + "/msxml6.dll")
+        this.wizard().wait(tr("Please wait while {0} is installed...", "msxml6"));
+        this.run(setupFile64, ["/q:a", "/c:msxml6_x64.msi /q"], null, false, true);
+    }
+    else {
```
Comment (ImperatorS79) [inline Engines/Wine/Verbs/msxml6/script.js]: Should not you also do the 32bits part for the amd64 case (i.e. put that part out of the if/else ?)

Comment (Zemogiter) [reply Engines/Wine/Verbs/msxml6/script.js]: I could but thats not how the wineprefix version acts.

Comment (ImperatorS79) [reply Engines/Wine/Verbs/msxml6/script.js]: ok, even winetricks does it that way.

**Pull request merged**